src/GuppyDataExplorer/GuppyDataExplorer.jsx

```diff
@@ -13,29 +13,98 @@ import {
   ChartConfigType,
 } from './configTypeDef';
 import './GuppyDataExplorer.css';
+import isEnabled from '../helpers/featureFlags';
 
 class GuppyDataExplorer extends React.Component {
   constructor(props) {
     super(props);
+    let initialState = {};
+    let initialFilter = {};
+
+    // aborting URL decode
+    if (isEnabled('storeExplorerStateInURL')) {
+      const stateFromURL = getQueryParameter('filters');
+      if (stateFromURL) {
+        const decodedFilter = atob(stateFromURL);
```

Comment:
suggest to also surround this with `try/catch` block

for example, if you delete a few chars from the base64 encoded filter string, paste it into address bar and enter, i think the page will crash because of this `atob()` will throw errors